aiohttp/web.py

```diff
@@ -421,7 +421,8 @@ def run_app(app, *, host=None, port=None, path=None, sock=None,
         loop.run_until_complete(app.shutdown())
         loop.run_until_complete(handler.shutdown(shutdown_timeout))
         loop.run_until_complete(app.cleanup())
-    loop.close()
+    if close_loop:
```

Comment:
I think there should be a check if loop was explicitly passed externally instead of any kind of a control flag.

Why? Because, if I pass the loop explicitly from outside, I control it from the outside and I have to take care about it from the outside, where I should close it whenever it's appropriate. If it's implicit loop created within the `run_app` function, then it's responsible to close it when it no longer need in it as it only owns it and nobody else.